src/core/Hash.zig

```diff
@@ -0,0 +1,48 @@
+const std = @import("std");
+const Sha256 = std.crypto.hash.sha2.Sha256;
+
+const base58 = @import("base58-zig");
+
+const Hash = @This();
```

Comment:
we're not a big fan of turning files into structs - its a very zig-only thing which will just be confusing for new devs and just another 'thing' to understand

Reply:
In my opinion that's an unfortunate tradeoff to make, but I understand.